Secret-agency-persistence/src/main/cz/fi/muni/pa165/secretagency/entity/Mission.java

```diff
@@ -113,11 +113,11 @@ public boolean equals(Object object) {
         if (!(object instanceof Mission)) return false;
 
         Mission mission = (Mission) object;
-        return java.util.Objects.equals(id, mission.getId()) &&
-                java.util.Arrays.equals(location, mission.getLocation()) &&
-                java.util.Objects.equals(missionType, mission.getMissionType()) &&
-                java.util.Objects.equals(started, mission.getStarted()) &&
-                java.util.Objects.equals(ended, mission.getEnded());
+        return Objects.equals(getId(), mission.getId()) &&
```

Comment:
Zatím bych id v equals a hashcode vynechal. Viz 
https://vladmihalcea.com/hibernate-facts-equals-and-hashcode/
https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/

Reply:
zmeniť location na 2 properties, vyhodiť ID z equals a hashcode